src/test/java/academy/softserve/edu/tests/TestMerchandiserAfterLogInPage.java

```diff
@@ -0,0 +1,60 @@
+package academy.softserve.edu.tests;
+
+import academy.softserve.edu.pageobjects.LogInPage;
+import academy.softserve.edu.pageobjects.MerchandiserOrderingPage;
+import academy.softserve.edu.pageobjects.UserInfoPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.concurrent.TimeUnit;
+
+public class TestMerchandiserAfterLogInPage {
+
+    public static final String LOG_IN_PAGE =
+            "http://192.168.56.101:8080/oms5/login.htm";
+
+    private WebDriver driver;
+    private UserInfoPage userInfoPage;
+
+    @DataProvider
```

Comment:
create a separate class for data providers - DataProviders.java, move it to utils. share this change with the team